src/utils/components/HorizontalNavbar/HorizontalNavbar.tsx

```diff
@@ -73,12 +81,14 @@ const HorizontalNavbar: FC<HorizontalNavbarProps> = ({ instanceTypeExpandedSpec,
           return (
             <Route
               Component={(props) => (
-                <Component
-                  instanceTypeExpandedSpec={instanceTypeExpandedSpec}
-                  obj={vm}
-                  params={params}
-                  {...props}
-                />
+                <StateHandler loaded={!expandedSpecLoading} withBullseye>
```

Comment:
how this works for VMs created with templates or YML?

Reply:
works great. THe expandedSpecLoading is `false` from the start with template vms